tests/gui/test_suggestions.py

```diff
@@ -14,8 +14,12 @@ def test_velocity_suggestions(centered_pair_predictions):
 
 def test_frame_increment(centered_pair_predictions: Labels):
 
-    # Testing videos that have less frames than desired Samples per Video value using stride method.
-    # Expected result is one suggested frame.
+    """
+    Testing videos that have less frames than desired Samples per Video (stride)
+
+    Expected result is there should be n suggestions where n is equal to the frames
+    in the video.
+    """
```

Comment:
Use docstrings for [function documentation](https://google.github.io/styleguide/pyguide.html#:~:text=A%20docstring%20is%20a%20string%20that%20is%20the%20first%20statement%20in%20a%20package%2C%20module%2C%20class%20or%20function.) only. Use [block comments](https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments) (just append a # in front of each line of code) here.